knowledge-base/StepsForAddingDataSource

```diff
@@ -0,0 +1,172 @@
+---
+title: Steps on how to add ObjectDataSource in a report designed in the Standalone Designer
+description: This KB article lists all of the required steps to add ObjectDataSource Compontent in the Standalone Designer.
+type: how-to
+page_title: Adding ObejectDataSource in the Standalone Designer
+slug: adding-obejectdatasiurce-in-the-standalone-designer
+position: 
+tags: 
+ticketid: 1407612
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Solution
+1. Open **Visual Studio** and create a new **Class Library**.
+2. Add the following piece of code from [How to: Bind to a BusinessObject article](../object-data-source-how-to-bind-to-business-object). Add the necessary Using-s. 
+
+```CSharp
```

Comment:
It is redundant to copy the same code from the conceptual topic. You can say that the users need to create their DLL file and just add a link to the conceptual topic for more details on how the code could look like.